artview/components/nav.py

```diff
@@ -63,19 +63,18 @@ def __init__(self, Vradar=None, Vgrid=None, Vfilelist=None,
         else:
             self.Vgrid = Vgrid
         if Vfilelist is None:
-            self.Vfilelist = Variable(0)
+            self.Vfilelist = Variable([])
         else:
             self.Vfilelist = Vfilelist
+        self.filename = ""
+        self.fileindex = 0
 
-        self.sharedVariables = {"Vradar": None,
-                                "Vgrid": None,
-                                "Vfilelist": None}
+        self.sharedVariables = {"Vradar": self.NewFile,
+                                "Vgrid": self.NewFile,
+                                "Vfilelist": self.NewFilelist}
```

Comment:
must map `NewFilelist` here or it will have no effect. Also rename removing the capital L for consistence with the variable name